packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2356,23 +2392,26 @@ export default class Meeting extends StatelessWebexPlugin {
    * @param {String} meetingInfo.body.locusUrl
    * @param {String} meetingInfo.body.sipUri
    * @param {Object} meetingInfo.body.owner
+   * @param {Object} destination locus object with meeting data
    * @returns {undefined}
    * @private
    * @memberof Meeting
    */
-  parseMeetingInfo(meetingInfo) {
+  parseMeetingInfo(meetingInfo, destination = null) {
     const webexMeetingInfo = meetingInfo?.body;
+    // We try to use as much info from Locus meeting object, stored in destination
+    const locusMeetingObject = typeof destination === 'object' ? destination : undefined;
 
     // MeetingInfo will be undefined for 1:1 calls
     if (webexMeetingInfo && !(meetingInfo.errors && meetingInfo.errors.length > 0)) {
-      this.conversationUrl = webexMeetingInfo.conversationUrl || this.conversationUrl;
-      this.locusUrl = webexMeetingInfo.locusUrl || this.locusUrl;
-      this.setSipUri(this.config.experimental.enableUnifiedMeetings ? webexMeetingInfo.sipUrl : webexMeetingInfo.sipMeetingUri || this.sipUri);
+      this.conversationUrl = locusMeetingObject?.conversationUrl || webexMeetingInfo.conversationUrl || this.conversationUrl;
```

Comment:
Do we have to use both locus.info and meetingInfo or can we skip one as the server are converting !?